extensions/adobe/experience/offer-management/domain-model.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/domain-model",
+  "meta:extensible": "true",
+  "type": "object",
+  "title": "Domain Model",
+  "description": "Domain Model",
+  "definitions": {
+    "offer": {
+      "properties": {
+        "xdm:schemaId": {
+          "type": "string",
+          "format": "uri",
+          "title": "Schema ID",
+          "description": "The unique identifier of the schema"
+        },
+        "xdm:name": {
+          "type": "string",
+          "title": "Name",
+          "description": "Model name"
+        },
+        "xdm:properties": {
```

Comment:
this doesn't seem like the right way to do properties.  There are some common patterns here that should be used.

Reply:
Now it's just an array of property names